server/models/cart.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const CartItemSchema = new Schema({
+   _id: { type: Schema.Types.ObjectId, ref: 'Product' },
+   quantity: Number,
+});
+
+const CartItem = mongoose.model('CartItem', CartItemSchema);
+

```

Comment:
you want to use cartitems as your table name for cart?
carts sounds good too. maybe look into that.

same with wishlist
(this is your personal preference)